dts/bindings/sensor/meas,ms56xx-spi.yaml

```diff
@@ -2,9 +2,9 @@
 # SPDX-License-Identifier: Apache-2.0
 
 description: |
-    TE Connectivity MS5607 digital pressure and temperature sensor.
+    TE Connectivity MS5607 and MS5611 digital pressure and temperature sensors.
     The Datasheet is at https://www.te.com/usa-en/product-CAT-BLPS0035.html
 
-compatible: "meas,ms5607"
+compatible: "meas,ms56xx"
```

Comment:
as you are renaming an existing compatible, it would require an entry in the migration guide to tell people what they need to change in their code that will now break. 
Another option would be to keep yaml files for the "old" ms5607 and have them include/inherit the new generic binding, to allow for backward compatibility